docs/source/livemode.md

```diff
@@ -0,0 +1,87 @@
+# Livemode - having a more familiar, possibly interactive, upgrade environment
+**Note that the livemode feature is experimental.**
```

Comment:
Note that the suggestion is littlebit malformed and not sure how to correctly input triple backticks here :) but you get the idea I believe.

```suggestion
```{warning}
Note that the livemode feature is experimental.
\`\`\`
```

Reply:
Suggestion applied to the `index.rst` of `experimental-features` subsection.